xml/selinux.xml

```diff
@@ -44,10 +47,10 @@
   <para>
    &selnx; was developed as an additional Linux security solution that uses
    the security framework in the Linux kernel. The purpose was to allow for
-   a more granular security policy that goes beyond what is offered by the
-   default existing permissions of Read, Write, and Execute, and beyond
-   assigning permissions to the different capabilities that are available on
-   Linux. &selnx; does this by trapping all system calls that reach the
+   a more granular security policy that goes beyond the standard 
+   Discretionary Access Controls (DAC), the traditional file permissions
+   of owner, group, world, and read, write, execute.
```

Comment:
IMHO this list of 6 entries with "and" only between the 3rd and 4th reads a little oddly.

Possible alternatives:

"of `owner`, `group`, and `world`, and of `read`, `write` and `execute`."

... or...

"of `owner`/`group`/`world` and `read`/`write`/`execute`."

... or just:

"of owner/group/world and read/write/execute."

Reply:
Imma like slashes.